org.adempiere.base/src/org/compiere/model/PO.java

```diff
@@ -126,6 +126,23 @@ public abstract class PO
 
 	/** default query/statement timeout, 300 seconds **/
 	private static final int QUERY_TIME_OUT = 300;
+	
+	/** Get value of the attribute for Table ID and Record ID  **/
+	private static final String	TABLE_ATTRIBUTE_VALUE_SQL = "SELECT a.Name, a.AttributeValueType, a.AD_Reference_ID, ta.Value, ta.ValueDate, ta.ValueNumber, ta.M_AttributeValue_ID "
+																		+ "	FROM AD_TableAttribute ta "
+																		+ "	INNER JOIN M_Attribute a ON (a.M_Attribute_ID = ta.M_Attribute_ID) "
+																		+ "	WHERE ta.AD_Table_ID = ? AND Record_ID = ? AND a.IsActive = 'Y' ";
+	/** Get Default value of the attribute **/
```

Comment:
Not a must, but for new code, it is better to use the """ text block instead of the old string concatenation approach.

Reply:
I think textblocks may not efficient as it uses more space as formatting is considered compared to concatation. As Java compiler normally at compile time treat this concatenations, there is no performance impact of concatenation so I think for sql concatenations is better.

Reply:
Performance and space efficiency difference would not be significant here (performance is debatable since string concatenation using + operator is generally consider not efficient), this is for readability and ease of maintenance (that's the key reason why Java added text block feature).

Reply:
@hengsin  converted to Text block and committed.